bazel.el

```diff
@@ -992,8 +1008,10 @@ strings.  Return either @WORKSPACE//PACKAGE:TARGET or
   "Search for a magic comment from point to BOUND.
 If a magic comment was found, return non-nil and set the match to
 the comment text."
-  (and (search-forward "keep sorted" bound t)
-       (nth 4 (syntax-ppss))))
+  (let ((case-fold-search t))
```

Comment:
Can you add a brief comment here explaining that Buildifier is case-insensitive here, maybe even with a reference?

Reply:
Done.  Reference points to `buildifier`'s source code since this behavior is undocumented.